js/package.json

```diff
@@ -26,16 +26,16 @@
   ],
   "scripts": {
     "build": "npm run build:lib && npm run build:dll && npm run build:app",
-    "build:app": "webpack --progress",
-    "build:lib": "webpack --config webpack.libraries --progress",
-    "build:dll": "webpack --config webpack.vendor --progress",
+    "build:app": "webpack --config webpack/config --progress",
```

Comment:
Could we maybe reanem the config now? Each are config files, we can call this one `app` (or something similar)